transaction_csv_cleanup.py

```diff
@@ -55,7 +55,7 @@ def get_files():
     b = g_config["input_filename"]
     c = g_config["fixed_prefix"]
     
-    if b is not "":
+    if b is not "" and g_config["path"] is not "":
```

Comment:
I don't understand what you want to happen here.

Reply:
This is meant to avoid an ugly error when no path is defined.